client/landing/gutenboarding/index.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import user from 'lib/user';
+import { main, redirectIfNotEnabled } from './controller';
+import { render as clientRender } from 'controller';
+import { wpDataDebugMiddleware } from './devtools';
+
+/**
+ * Style dependencies
+ */
+import 'assets/stylesheets/gutenboarding.scss';
+import 'components/environment-badge/style.scss';
```

Comment:
I would expect this style to be applied in every CSS entry point always, and not needing to summon it separately. Good as-is for now tho but just something to maybe refactor in future. cc @jsnajdr

Reply:
The `EnvironmentBadge` component is special, and might be a good candidate for completely inlining its styles. Either in the `index.html` document returned from server, or as CSS-in-JS.

Reply:
Yup, sounds still good for another PR. It also needs little more styling work to look sharp without Calypso base styles:

https://github.com/Automattic/wp-calypso/pull/37141#issuecomment-548256106